assets/src/settings-page/sandboxing.js

```diff
@@ -0,0 +1,120 @@
+import { STANDARD } from '../common/constants';
+
+/**
+ * Component rendering the Sandboxing experiment.
+ *
+ * @param {Object} props                Component props.
+ * @param {string} props.focusedSection Focused section.
+ */
+export function Sandboxing( { focusedSection } ) {
+	const { updateOptions, editedOptions: {
+		sandboxing_enabled: sandboxingEnabled,
+		sandboxing_level: sandboxingLevel,
+		theme_support: themeSupport,
+	} } = useContext( Options );
+
+	if ( STANDARD !== themeSupport ) {
+		return null;
+	}
+
+	return (
+		<AMPDrawer
+			heading={ (
+				<h3>
+					{ __( 'Sandboxing (Experimental)', 'amp' ) }
+				</h3>
+			) }
+			hiddenTitle={ __( 'Sandboxing (Experimental)', 'amp' ) }
+			id="sandboxing"
+			initialOpen={ 'sandboxing' === focusedSection }
+		>
+			<p>
+				{ __( 'Try out a more flexible AMP by generating pages that use AMP components without requiring AMP validity! By selecting a sandboxing level, you are indicating the minimum degree of sanitization. For example, if you selected the loose leve but have a page without any POST form and no custom scripts, it will still be served as valid AMP—the same as if you had selected the strict level.', 'amp' ) }
+			</p>
+
+			<CheckboxControl
+				className="sandboxing-enabled"
+				checked={ sandboxingEnabled }
+				label={ __( 'Enable sandboxing experiment', 'amp' ) }
+				onChange={
+					( newChecked ) => {
+						updateOptions( { sandboxing_enabled: newChecked } );
+					}
+				}
+			/>
+
+			{ sandboxingEnabled && (
+				<ol>
+					<li>
+						<input
+							type="radio"
+							id="sandboxing-level-1"
+							checked={ 1 === sandboxingLevel }
+							onChange={ () => {
+								updateOptions( { sandboxing_level: 1 } );
+							} }
+						/>
+						<label htmlFor="sandboxing-level-1">
+							<strong>
+								{ __( 'Loose:', 'amp' ) }
+							</strong>
+							{ ' ' + __( 'Do not remove any AMP-invalid markup by default, including custom scripts. CSS processing is disabled.', 'amp' ) }
+						</label>
+					</li>
+					<li>
+						<input
+							type="radio"
+							id="sandboxing-level-2"
+							checked={ 2 === sandboxingLevel }
+							onChange={ () => {
+								updateOptions( { sandboxing_level: 2 } );
+							} }
+						/>
+						<label htmlFor="sandboxing-level-2">
+							<strong>
+								{ __( 'Moderate:', 'amp' ) }
+							</strong>
+							{ ' ' + __( 'Remove anything invalid AMP except for POST forms, excessive CSS, and other PX-verified markup.', 'amp' ) }
+						</label>
+					</li>
+					<li>
+						<input
+							type="radio"
+							id="sandboxing-level-3"
+							checked={ 3 === sandboxingLevel }
+							onChange={ () => {
+								updateOptions( { sandboxing_level: 3 } );
+							} }
+						/>
+						<label htmlFor="sandboxing-level-3">
+							<strong>
+								{ __( 'Strict:', 'amp' ) }
+							</strong>
+							{ ' ' + __( 'Require valid AMP, removing all markup that causes validation errors (except for excessive CSS).', 'amp' ) }
```

Comment:
How about using the `createInterpolateElement` so that the string offers more context for translators, i.e.:

```suggestion
							{ createInterpolateElement(
								__( '<b>Strict:</b> Require valid AMP, removing all markup that causes validation errors (except for excessive CSS).', 'amp' ),
								{ b: <strong /> },
							) }
```